master/buildbot/buildslave/__init__.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, name, password, max_builds=None,
         self.password = password
         self.friendly_name = friendlyName
         self.eid = eid  # External ID
+        self.fqdn = fqdn
```

Comment:
Could we have the more descriptive variable name here? And modify documentation.